tajo-core-tests/src/test/java/org/apache/tajo/engine/query/TestTablePartitions.java

```diff
@@ -1542,6 +1542,19 @@ public final void testDatePartitionColumn() throws Exception {
     assertEquals(expectedResult, resultSetToString(res));
     res.close();
 
+    // IN
+    res = executeString("SELECT * FROM " + tableName
+      + " WHERE key IN ( to_date('1994-02-02', 'YYYY-MM-DD'), to_date('1993-11-09', 'YYYY-MM-DD')) order by col1, " +
```

Comment:
SQL standard representation for date would be simpler; e.g., ``DATE '1994-02-02'``.